docker/Dockerfile.release

```diff
@@ -56,15 +56,30 @@ RUN pip3 install -U pip
 WORKDIR /opt
 
 # CMake
-RUN CMAKE_VERSION=3.17 && \
-    CMAKE_BUILD=3.17.4 && \
+RUN CMAKE_VERSION=3.18 && \
+    CMAKE_BUILD=3.18.6 && \
     wget -nv https://cmake.org/files/v${CMAKE_VERSION}/cmake-${CMAKE_BUILD}.tar.gz && \
     tar -xf cmake-${CMAKE_BUILD}.tar.gz && \
     cd cmake-${CMAKE_BUILD} && \
     ./bootstrap --parallel=$(grep ^processor /proc/cpuinfo | wc -l) -- -DCMAKE_USE_OPENSSL=OFF && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install && \
     rm -rf /cmake-${CMAKE_BUILD}
 
+# Miniconda
+ENV PATH="/opt/conda/bin:${PATH}"
+RUN MINICONDA_URL="https://repo.anaconda.com/miniconda/Miniconda3-py38_4.12.0-Linux-x86_64.sh"; \
+    SHA256SUM="3190da6626f86eee8abf1b2fd7a5af492994eb2667357ee4243975cdbb175d7a"; \
+    wget "${MINICONDA_URL}" -O miniconda.sh -q && \
+    echo "${SHA256SUM} miniconda.sh" > shasum && \
+    sh miniconda.sh -b -p /opt/conda && \
+    rm miniconda.sh shasum && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc && \
+    find /opt/conda/ -follow -type f -name '*.a' -delete && \
+    find /opt/conda/ -follow -type f -name '*.js.map' -delete && \
+    /opt/conda/bin/conda clean -afy
```

Comment:
What is the purpose of this set of commands?

Reply:
Good question. The first two I believe remove some build/install artifacts. The last one, from the conda help:

```
$ conda clean --help

Remove unused packages and caches.

  -a, --all             Remove index cache, lock files, unused cache packages, and tarballs.
  -f, --force-pkgs-dirs
                        Remove *all* writable package caches. This option is not included with the --all flag. WARNING: This will break environments with packages installed using symlinks back to the package cache.
  -y, --yes             Do not ask for confirmation.
```